IO/src/rtmp/CMakeLists.txt

```diff
@@ -105,6 +105,23 @@ endif(RTMP_QUICKSYNC)
 
 
 if(RTMP_NVENC)
+
+if(LINUX)
```

Comment:
Just noticed, the whole block needs to be indented by two spaces (it's inside `if(RTMP_NVENC)`).

Reply:
Done.

Reply:
🙇